src/components/__tests__/Link-test.js

```diff
@@ -28,42 +46,33 @@ describe('Link', () => {
     }
 
     it('dispatches a `changeLocation` action', () => {
-      const dispatch = sinon.spy()
-      const context = { store: { dispatch } }
-      const wrapper = shallow(
-        <Link to={{ pathname: '/' }} />,
-        { context },
-      )
-      wrapper.simulate('click', event)
+      const wrapper = setup({ to: { pathname: '/' } })
+      wrapper.find('Link').simulate('click', event)
       expect(dispatch).to.have.been.calledWith(push('/'))
     })
 
     it('handles a `replace` prop', () => {
-      const dispatch = sinon.spy()
-      const context = { store: { dispatch } }
-      const wrapper = shallow(
-        <Link to={{ pathname: '/' }} replace />,
-        { context },
+      const wrapper = setup(
+        { to: { pathname: '/' }, replace: true }
       )
-      wrapper.simulate('click', event)
+      wrapper.find('Link').simulate('click', event)
       expect(dispatch).to.have.been.calledWith(replace('/'))
     })
 
     it('handles a `onClick` prop', () => {
       const onClick = sinon.spy()
-      const wrapper = shallow(<Link to={{ pathname: '/' }} onClick={onClick} />)
-      wrapper.simulate('click', event)
-      expect(onClick).to.have.been.calledWith(event)
+      const wrapper = setup({ to: { pathname: '/' }, onClick })
+      wrapper.find('Link').simulate('click', event)
+      const calledWithSyntheticEvent = onClick
+        .args[0][0]
+        .constructor
+        .name === 'SyntheticEvent'
+      expect(calledWithSyntheticEvent).to.equal(true)
```

Comment:
i am really not sure why this stopped working as it was written before. the event handler has not changed at all